EnTKdevel/README.md

```diff
@@ -1,5 +1,117 @@
-## Introduction
+# Introduction
 
-This README contains some simple instructions how to setup the environment to execute the EnTK pipeline notebook or script. 
+This README file contains basic instructions on how to execute the Ensemble Toolkit 
+script for this use case
 
-Please prefer to use the script compared to the notebook, until the pipeline is completely stable. 
\ No newline at end of file
+It is suggested to read [RADICAL-Pilot's documentation](https://radicalpilot.readthedocs.io/en/latest/)
+and [Ensemble Toolkit's documentation](https://radicalentk.readthedocs.io/en/latest/).
+
+It is also recommended that the execution is done from a Virtual Machine that has constant 
+Internet connection, Docker, and runs some version of Ubuntu.
+
+## Preparing your environment for installation
+
+Verify that you have a Python 2.7 installation and you can create a virtual environment
+by executing `virtualenv` or `conda create -n test`. Please do not create a virtual environment now.
+
+If you are not sure, please install [Miniconda 2](https://conda.io/miniconda.html) before proceeding.
+In either case, instructions are provided for both GCC python and Conda Python.
+
+The next step would be to install Rabbit MQ, since it is needed from Ensemble Toolkit:
+
+```
+~$ docker run -d --name entk_queues -P rabbitmq:3
+~$ docker ps
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                                                                                 NAMES
+d334b23418f2        rabbitmq:3          "docker-entrypoint..."   3 months ago        Up 3 months         0.0.0.0:32775->4369/tcp, 0.0.0.0:32774->5671/tcp, 0.0.0.0:32773->5672/tcp, 0.0.0.0:32772->25672/tcp   entk_queues
+```
+Verify that there is a port that looks like `0.0.0.0:32772->25672/tcp`. The second part should be the same.
+
+After RabbitMQ is installed, we need to install gsissh and my-proxy. Instructions
+can be found [here](https://github.com/vivek-bala/docs/blob/master/misc/gsissh_setup_stampede_ubuntu_xenial.sh)
```

Comment:
I think this link needs to be updated, there are some commands that do not work.

Reply:
Let me check with @vivek-bala to see with we can do. Although that is usually trial and error.